### DISCUSSION TITLE
RaspberryMatic als Hass.IO Addon: Keine Events mehr nach Update auf aktuelle Version

### DISCUSSION THREAD

**Discussion by northnorway** [OpenCCU/OpenCCU#1740] [🇩🇪 Fragen&Antworten]
Hallo! Ich bin vor einigen Wochen auf das RaspberryMatic Plugin umgestiegen und benutze aktuell einen 4-Tasten Handsender. Tastendrücke werden ausgewertet, indem ich mit NodeRed auf Events vom Typ "homematic.keypress" reagiere. Damit diese Events korrekt von RaspberryMatic gesendet werden, habe ich für jede Taste ein Programm in RaspberryMatic angelegt. Dies hat einwandfrei funktioniert, bis zum Update des RaspberryMatic Addons  auf die aktuelleste Version (3.61.7.20220226 ). 
Nun bin ich ratlos, wie ich die gewohnte Funktionalität wieder herstellen kann und wäre für jeden Hinweis äußerst dankbar!
Im Voraus vielen Dank!

**Comment by Baxxy13** [comment]
Hast du mal getestet ob es geht wenn du NodeRed direkt mit dem RM-AddOn verbindest?
Wie das geht hatte ich [hier](https://github.com/jens-maus/RaspberryMatic/discussions/1703#discussioncomment-2164197) beschrieben.

**Reply by northnorway** [reply]
Danke für den Hinweis auf diese Möglichkeit. Leider ist es mir nicht spontan gelungen, auf diese Weise mit RaspberryMatic zu kommunizieren. Und um ehrlich zu sein, finde ich die Möglichkeit über "normale" Events zu gehen auch etwas charmanter, weil dies eben der Standard in Home Assistant ist und man somit auch außerhalb von NodeRed agieren kann. 
Mein Problem bleibt also weiterhin ungelöst: Nach dem Update kommen keine Events mehr in Home Assistant an. Freue mich anch wie vor über Hilfestellungen!

**Reply by northnorway** [reply]
Das Problem hat sich nun durch ein weiteres Update ebenso schnell gelöst wie es gekommen ist: Seit dem Update von Home Assistant Core auf die Version core-2022.3.1 kommen die Nachrichten wieder an und können ausgewertet werden. Problem ist somit gelöst.
